bodhi/tests/server/scripts/test_approve_testing.py

```diff
@@ -129,13 +127,12 @@ def test_non_autokarma_critpath_update_meeting_karma_requirements_gets_one_comme
         update.status = models.UpdateStatus.testing
         update.comment(self.db, u'testing', author=u'hunter2', anonymous=False, karma=1)
 
-        with patch(
-                'bodhi.server.scripts.approve_testing._get_db_session', return_value=self.db):
-            approve_testing.main(['nosetests', 'some_config.ini'])
-
-            # Now we will run main() again, but this time we expect Bodhi not to add any further
-            # comments.
-            approve_testing.main(['nosetests', 'some_config.ini'])
+        with patch('bodhi.server.scripts.approve_testing.initialize_db'):
+            with patch('bodhi.server.scripts.approve_testing.get_appsettings', return_value=''):
+                approve_testing.main(['nosetests', 'some_config.ini'])
+                # Now we will run main() again, but this time we expect Bodhi not to add any
```

Comment:
It's a small detail, but I have a testing style where I use spacing to try to create three overall blocks in my tests: setup, line that calls thing I'm testing, and then assertions. This test actually had four since the with statement would make flake8 mad, but I actually like the space between this line and the one above, because this test is actually asserting that the second call to approve_testing works correctly (it assumes the one above does work correctly).

Same in other places in this PR.